components/Audio.tsx

```diff
@@ -1,7 +1,7 @@
 import "vidstack/styles/defaults.css"
 import "vidstack/styles/community-skin/audio.css"
 
-import { alignClasses } from "@utils/puckUtils"
```

Comment:
Any specific reason why you changed filename?

Reply:
To group things about `puck` in one folder.